### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by Expertium**
I suggest you to take a close look at "green shades, red shades", specifically, the < > signs and threshold values for small/medium/large, they seem to be inconsistent

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,22 +134,22 @@ You may have noticed that FSRS-5 has a 99.0% superiority over SM-2, meaning that
...
 2. Paired t-test Cohen's d values (effect sizes)
 
 For both visualizations, the colors indicate:
 
 - Red shades indicate the row algorithm performs worse than the column algorithm:
-  - Dark red: Large effect (r > 0.5 or d > 0.5)
-  - Red: Medium effect (0.2 < r/d ≤ 0.5) 
-  - Light red: Small effect (r/d ≤ 0.2)
+  - Dark red: large effect (r > 0.5 or d > 0.5)
+  - Red: medium effect (0.5 < r or d ≤ 0.5)
```
Comment (L-M-Sherlock) [inline README.md]: Both effect sizes are between 0.2 and 0.5 here.

**Review by L-M-Sherlock — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Expertium) [reply README.md]: I can't find the file that you used to make these tables. If you pointed me to it, I could just look at the code to figure out what exactly you were doing and how to write about it.
EDIT: nvm, I found it https://github.com/open-spaced-repetition/srs-benchmark/blob/9c89d1e97c77b058168c3ebab27acec524d87ef3/significance_table.py#L150

**Comment by Expertium** [comment]
I can't fix the formatting here
image

**Review by L-M-Sherlock — APPROVED** [review]
LGTM

**Pull request merged**